lib/prefix.c

```diff
@@ -556,6 +556,31 @@ int prefix_match(const struct prefix *n, const struct prefix *p)
 	return 1;
 }
 
+/* If n includes p prefix then return 1 else return 0. */
+int prefix_opaque_match(const struct prefix *n, const struct prefix *p)
```

Comment:
We should call prefix_match() and check for appropriate family and do the right thing instead of having a different special call.

Reply:
done